conda/environments/cuml_dev_cuda11.0.yml

```diff
@@ -26,8 +26,8 @@ dependencies:
 - pip
 - pip:
     - sphinx_markdown_tables
-    - git+https://github.com/dask/dask.git@main
-    - git+https://github.com/dask/distributed.git@main
+    - git+https://github.com/dask/dask.git@2021.05.1
+    - git+https://github.com/dask/distributed.git@2021.05.1
```

Comment:
Seeing similar lines in `cuml_dev_cuda10.1.yml` and `cuml_dev_cuda10.2.yml`. Should we worry about those or do they not matter?

Reply:
They are not used since we require cuda 11+ and we’re just forgotten to be removed in 21.06

Reply:
Sounds good. Feel free to mark resolved (appears I can't do that)